packages/react-i18n/src/i18n.ts

```diff
@@ -403,9 +403,13 @@ export class I18n {
       ...options,
     }).format(amount);
 
-    return shortSymbol.prefixed
+    const formattedWithSymbol = shortSymbol.prefixed
       ? `${shortSymbol.symbol}${formattedAmount}`
       : `${formattedAmount}${shortSymbol.symbol}`;
+
+    return amount < 0
+      ? `-${formattedWithSymbol.replace('-', '')}`
```

Comment:
Are there any possible formats of `formattedWithSymbol` where a `-` would denote something that is not a minus symbol? I see `1 234,56 Kč CZK`, could there ever be a format of money that would result in `1 234,56 foo-bar`

Reply:
I think we're "safe" to do this.
 - We should always have `-` as the `minus sign`.  Looking at the Unicode CLDR data, every single locale for which it defines a `minus_sign:` uses `-`
  - We should never have a `-` in this string that isn't a `minus sign`.  We're dealing with the "symbol" for the currency, or potentially the ISO-4217 currency code (which is guaranteed to be three alphabetic letters).

It's possible that a `-` might show up in the full-text name of the currency (e.g. `Guinea-Bisseau franc`), but we won't have the full name here, only the symbol or the three-letter code.